includes/Modules/Analytics_4.php

```diff
@@ -714,27 +707,25 @@ protected function get_datapoint_definitions() {
 			),
 		);
 
-		if ( Feature_Flags::enabled( 'audienceSegmentation' ) ) {
-			$datapoints['POST:create-audience']                      = array(
-				'service'                => 'analyticsaudiences',
-				'scopes'                 => array( self::EDIT_SCOPE ),
-				'request_scopes_message' => __( 'You’ll need to grant Site Kit permission to create new audiences for your Analytics property on your behalf.', 'google-site-kit' ),
-			);
-			$datapoints['POST:save-resource-data-availability-date'] = array(
-				'service' => '',
-			);
-			$datapoints['POST:sync-audiences']                       = array(
-				'service'   => 'analyticsaudiences',
-				'shareable' => true,
-			);
-			$datapoints['GET:audience-settings']                     = array(
-				'service'   => '',
-				'shareable' => true,
-			);
-			$datapoints['POST:save-audience-settings']               = array(
-				'service' => '',
-			);
-		}
+		$datapoints['POST:create-audience']                      = array(
```

Comment:
We can now include these datapoints in the `$datapoints` array at the point of its definition. Again, I'll apply this change to save the round trip.